nix/pkgs/haskell/haskell.nix

```diff
@@ -122,6 +203,19 @@ let
             platforms = lib.platforms.linux;
           };
 
+          marlowe-actus.components.exes.marlowe-actus-test-kit = {
+            build-tools = r-packages;
+            # Seems to be broken on darwin for some reason
+            platforms = lib.platforms.linux;
+          };
+
+          # Windows build of libpq is marked as broken
```

Comment:
Would putting the `if os(windows) buildable: false` conditional in cabal files fix these also?

Reply:
libpq is only broken for windows cross compile with nix, but it might build on windows itself.

Reply:
Right, but what you're putting here is *also* saying it will only build on linux or darwin. So while it may be overly zealous to say that `fake-pab` etc won't build on windows, I don't think it's too bad.

Reply:
I guess I felt the restriction belonged in nix because libpq was likely only broken in nix.  We can put it in the `.cabal` file, but it will prevent someone building `fake-pab` etc. on windows (even if they have working libpq installed).  Is `fake-pab` something `cardano-node` might use for testing on their windows github builders in the future?

Reply:
Right now, the only library that we actually care about *any* kind of windows support for is `plutus-ledger-api`. That's the thing that `cardano-node` depends on. I know people will be cross-compiling that and maybe even native-compiling it.

Everything else is gravy right now. Maybe later we'll need to revisit this, but for now I'm mostly in favour of doing the simplest and most maintainable thing.

Reply:
still relevant